tests/migrator/test_journals.py

```diff
@@ -252,7 +252,39 @@ def test_migration(app):
                         {
                             "subscription": "v 1 (1992) -",
                             "url": "https://www.radioeng.cz/search.htm",
-                            "access_type": "6",
+                            "access_type": ["OPEN_ACCESS"],
+                            "note": "tada",
+                        }
+                    ],
+                    "items": [],
+                },
+            },
+        )
+        check_transformation(
+            """
+            <datafield tag="856" ind1="4" ind2="1">
+                <subfield code="3">v 1 (1992) -</subfield>
+                <subfield code="u">
+                https://www.radioeng.cz/search.htm
+                </subfield>
+                <subfield code="x">789</subfield>
+                <subfield code="z">tada</subfield>
+            </datafield>
+            """,
+            {
+                "_migration": {
+                    "is_multipart": False,
+                    "has_related": False,
+                    "related": [],
+                    "record_type": "journal",
+                    "volumes": [],
+                    "electronic_items": [
+                        {
+                            "subscription": "v 1 (1992) -",
+                            "url": "https://www.radioeng.cz/search.htm",
+                            "access_type": ["RESTRICTED_PERPETUAL_ACCESS_BACKFILES",
```

Comment:
should be a list, did you find example of records which have more than one number in `85641x ` ?

Reply:
I checked, and it happens indeed